cli/src/commands/debug/copy_detection.rs

```diff
@@ -20,8 +20,9 @@ use std::path::{Path, PathBuf};
 
 use futures::executor::block_on_stream;
 use futures::StreamExt;
-use jj_lib::backend::{Backend, CopyRecord, CopySource, CopySources};
+use jj_lib::backend::{Backend, CopyRecord};
 use jj_lib::default_index::{AsCompositeIndex as _, DefaultIndexStore};
+use jj_lib::matchers::EverythingMatcher;
```

Comment:
nit: `EverythingMatcher` is unused (but unchecked because of `allow(unused, dead_code)`.)

Reply:
there was a lot of unused stuff being supressed there.  Good catch.